images/k8sgpt/tests/main.tf

```diff
@@ -10,5 +10,5 @@ variable "digest" {
 
 data "oci_exec_test" "runs" {
   digest = var.digest
-  script = "${path.module}/01-runs.sh"
+  script = "docker run --rm $IMAGE_NAME --help"
```

Comment:
Hmm, I've been using `$$IMAGE_NAME`, so we may have some cleanup to do

Reply:
Yep that's right.